src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -43,7 +44,7 @@ async function boot(options?: Partial<WebAssemblyStartOptions>): Promise<void> {
     }
   });
 
-  Blazor._internal.InputFile = WasmInputFile;
+  Blazor._internal.InputFile = InputFile;
```

Comment:
If the value for this is always the same on all hosting models now, could we move the assignment logic into `GlobalExports.ts` (like how it assigns `Blazor._internal.Virtualize`) so it doesn't have to be set separately in all three `Boot.*.ts` files?

Reply:
Might be possible to do that for `Blazor._internal.getJSDataStreamChunk` too if the value is the same everywhere (even if it's not strictly needed on `Blazor._internal.*` on Server, since we do still use the code from the function on Server, so it's not like the js bundle would be getting any bigger)